ktor-server/ktor-server-plugins/ktor-server-openapi/jvm/src/io/ktor/server/openapi/OpenAPI.kt

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2014-2022 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.openapi
+
+import io.ktor.server.application.*
+import io.ktor.server.html.*
+import io.ktor.server.http.content.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import io.swagger.codegen.v3.*
+import io.swagger.codegen.v3.generators.html.*
+import io.swagger.parser.*
+import io.swagger.v3.parser.core.models.*
+import kotlinx.html.*
+import java.io.*
+
+/**
+ * Create a get endpoint with [SwaggerUI] rendered from open api file at [filename].
+ */
+public fun Routing.swaggerUI(path: String, filename: String) {
```

Comment:
Parameter `path` is confusing. At first, I thought that it's a path to swagger file. Can you write api doc for parameters?

Reply:
Thanks, will add. Could you tell me if you have any name ideas?

Reply:
I think "path" is ok, because it's consistent with other routing methods. With API doc it should be clear